patch/guides (#270)
nodejs/i18n: README.md

> # Node.js i18n

Reviewer: I would more closely match the GitHub banner for the repo:

> The Node.js Internationalization (i18n) Working Group

[![Crowdin](https://badges.crowdin.net/nodejs/localized.svg)](https://crowdin.com/project/nodejs)

> Here you can find contribution guides, discuss ideas or join meetings. Here also contains original and translated Node.js API documentation.

Reviewer 2: ```suggestion
Here you can find contribution guides, discuss ideas or join meetings. This repo also contains the source English Node.js API documentation as well as translated versions of the documentation. The source English is copied on a schedule from its canonical home in the [doc](https://github.com/nodejs/node/tree/master/doc) directory of the nodejs/node repository. The translated files are added by pull requests from our Crowdin integration.
```

> This repo is a basement of Node.js projects internationalization initiative. Here you can find contribution guides, discuss ideas or join meetings. Here also contains original and translated Node.js API documentation.

Reviewer: 'basement' is not idiomatic here (though I understand your meaning), I think you can just delete this sentence because it's in the opening line:

~This repo is a basement of Node.js projects internationalization initiative.~

## Contributing

> Help us get Node.js translated into your native language and reach more people's hearts. Please read next guides before:

Reviewer 2: ```suggestion
Help us get Node.js translated into your native language and reach more people's hearts. Please read these guides to get started:
```

> Help us get the Node.js translated into your native language and reach more people's hearts. Please read next guides before:

Reviewer: Either drop 'the':
> ~the~ Node.js
… or add project
> the Node.js *project*

Reviewer 2: .. or `the Node.js project documentation`

Author: This text related not only to documentation, so I just dropped `the`

| Yuki Okushi         | [JohnTitor](https://github.com/JohnTitor)                 | [JustM0nik4](https://twitter.com/JustM0nik4)             |
| Zeke Sikelianos     | [zeke](https://github.com/zeke)                           | [zeke](https://twitter.com/zeke)                         |

## Joining

> If you're interested in joining this group, or would like to leave a question or comment for its members - please [create an issue](https://github.com/nodejs/i18n/issues/new) or submit a pull request. Keep in mind, you don't have to be a member of this group to start translating, more info about translators contribution read in [get started](./guides/GET_STARTED.md) guide.

Reviewer: Change to:
> You can read more about contributing translations in the [Getting Started] guide

> If you're interested in joining this group, or would like to leave a question or comment for its members - please [create an issue](https://github.com/nodejs/i18n/issues/new) or submit a pull request. Keep in mind, you don't have to be a member of this group to start translating. You can read more about contributing translations in the [Getting started](./guides/GETTING_STARTED.md) guide.

Reviewer 2: ```suggestion
If you're interested in joining this group or would like to leave a question or comment for its members, please [create an issue](https://github.com/nodejs/i18n/issues/new) or submit a pull request. Keep in mind, you don't have to be a member of this group to start translating. You can read more about contributing translations in the [Getting started](./guides/GETTING_STARTED.md) guide.
```
